client/landing/site.landing/coffee/team/forms/teamdomaintabform.coffee

```diff
@@ -19,42 +19,34 @@ module.exports = class TeamDomainTab extends kd.FormView
       click        : => @input.setFocus()
 
     @inputView.addSubView @input = new kd.InputView
-      placeholder  : 'your-team'
-      attributes   : 10
+      placeholder  : 'Your team name...'
```

Comment:
placeholder is done in a url format intentionally even if we've decided to change it please change it so, e.g. `your-team-name`